tests/run-pass/aux_test.rs

```diff
@@ -1,4 +1,5 @@
 // aux-build:dep.rs
+// This ignores the test against rustc, but runs it against miri:
```

Comment:
Hm, you are right. I am sure I used to see that test fail. But anyway, seems to be fixed now.